docs/content/latest/benchmark/ycsb-ycql.md

```diff
@@ -98,3 +98,8 @@ To run the other workloads (for example, `workloadb`), all we need to do is chan
 ```sh
 $ ./bin/ycsb run yugabyteCQL -P yugabyteCQL/db.properties -P workloads/workloadb
 ```
+
+{{< note title="Note" >}}
+Within the tar-ball there is a handy script (run_cql.sh), that runs all the workloads for you. You would need to just specify the paths to the binary and the proper IPs.
```

Comment:
Lets add this to the beginning itself since this seems to be a more convenient way to run the benchmark.

Flow can be:
1. Download the YCSB binaries
2. Start yugabytedb
3. Run the workload using `run_cql.sh`

Reply:
Done.